websites/A/Aniyan/presence.ts

```diff
@@ -8,51 +8,47 @@ const presence = new Presence({
 
 let lastPlaybackState = null,
   playback,
-  browsingTimestamp = Math.floor(Date.now() / 1000);
+  browsingStamp = Math.floor(Date.now() / 1000);
 
 if (lastPlaybackState !== playback) {
   lastPlaybackState = playback;
-  browsingTimestamp = Math.floor(Date.now() / 1000);
+  browsingStamp = Math.floor(Date.now() / 1000);
 }
 
 presence.on("UpdateData", async () => {
-  playback =
-    document.querySelector("#player > div.jw-media.jw-reset > video") !== null
-      ? true
-      : false;
+  const video: HTMLVideoElement = document.querySelector(
+    "#player > div.jw-media.jw-reset > video"
+  );
+
+  playback = video !== null ? true : false;
```

Comment:
```suggestion
  playback = video ? true : false;
```

Reply:
the playback variable is used somewhere in the top part of the code, I am not removing it.